planet/models.py

```diff
@@ -265,6 +266,29 @@ def _get_random_filename(content_type=None):
     return name
 
 
+@define
+class OrderComponent:
+    delivery: str
+    expires: datetime
```

Comment:
@jreiberkyle the API response has `expires_at`, but that's an idiosyncracy. I think users will be less surprised to see `created` since that's a standard metadata term: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/.